brianlong/www.validators.app
# 182783470 pipeline blacklist

#### What's this PR do?
- Add blacklist for validators that should be omitted by the pipeline

#### How should this be manually tested?
- run tests
- run gather_rpc_testnet_daemon.rb and see if there's no errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182783470)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
